src/main.rs

```diff
@@ -262,3 +264,220 @@ mod test {
         assert_eq!(expected, actual);
     }
 }
+
+#[cfg(test)]
+mod cli_test {
```

Comment:
I moved the cli tests here to be able to use the `get_mock_server` helper

Reply:
Ok, that breaks `cargo test` when running these tests from clean slate.

I will revert and explore ways of sharing the helper.

Reply:
After struggling with this a bit, I think it will be hard to do without introducing a library.

I ended up duplicating these functions :disappointed: -- I think we should re-visit this after #46 is merged.

Reply:
Yeah no worries.